CMSgov/onemac
# OY2-17922: Create Submission Form: 1915(b) Waiver Amendment for Waivers already in OneMAC

Story: https://qmacbis.atlassian.net/browse/OY2-17922
Endpoint: See github-actions bot comment

### Details
We want to be able to select an approved initial or renewal waiver to amend right from the dashboard and details pages.

### Changes
- Created the Add Amendment action
- Associated the action with inital and renewal waivers in Approved status
- Updated the form to insert the parent id into the correct component id field

### Test Plan
1. Login as a submitting user
2. Navigate to Package View/Waivers tab
3. Click the Action kebab on an initial waiver in approved status
4. Verify Add Amendment is an action option in the popup
5. Click action and verify navigation to the Waiver Amendment form with the waiver number filled in 
6. Verify editing of that field is disabled
7. Complete the form and submit the amendment
8. Navigate to Package View/Waivers tab
9. Click the ID of an initial waiver in approved status
10. Verify Add Amendment is an action option in the Package Actions list
11. Click action and verify navigation to the Waiver Amendment form with the waiver number filled in 
12. Verify editing of that field is disabled
13. Complete the form and submit the amendment
